db/migrate/20200730100922_add_foreign_key_constraint_to_active_storage_attachments_for_blob_id.active_storage.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from active_storage (originally 20180723000244)
+class AddForeignKeyConstraintToActiveStorageAttachmentsForBlobId < ActiveRecord::Migration[6.0]
+  def up
+    return if foreign_key_exists?(:active_storage_attachments, column: :blob_id)
+
+    if table_exists?(:active_storage_blobs)
+      add_foreign_key :active_storage_attachments, :active_storage_blobs, column: :blob_id
+    end
+  end
```

Comment:
Maybe it helps to also define the `down` method in case you want to rollback

Reply:
@Nechitadi definitely :D

Reply:
Will do, once I have any idea what that method should do and how does it work. For now, I'll leave it on TODO :)